packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -57,6 +56,154 @@ class NetworkController extends DisposableController
     subscribeToFilterChanges();
   }
 
+  String? exportAsHarFile() {
```

Comment:
Can you add some documentation? It'd probably be good to link to the specification for the HAR format as well.

Reply:
Added doc comments